src/collective/volto/formsupport/browser/templates/email_confirm_view.pt

```diff
@@ -0,0 +1,362 @@
+        opacity: 0;
+        overflow: hidden;
+        mso-hide: all;
+        visibility: hidden;
+        width: 0;
+      }
+
+      .powered-by a {
+        text-decoration: none;
+      }
+
+      hr {
+        border: 0;
+        border-bottom: 1px solid #f6f6f6;
+        margin: 20px 0;
+      }
+
+      /* -------------------------------------
+          RESPONSIVE AND MOBILE FRIENDLY STYLES
+      ------------------------------------- */
+      @media only screen and (max-width: 620px) {
+        table.body h1 {
+          font-size: 28px !important;
+          margin-bottom: 10px !important;
+        }
+        table.body p,
+        table.body ul,
+        table.body ol,
+        table.body td,
+        table.body span,
+        table.body a {
+          font-size: 16px !important;
+        }
+        table.body .wrapper,
+        table.body .article {
+          padding: 10px !important;
+        }
+        table.body .content {
+          padding: 0 !important;
+        }
+        table.body .container {
+          padding: 0 !important;
+          width: 100% !important;
+        }
+        table.body .main {
+          border-left-width: 0 !important;
+          border-radius: 0 !important;
+          border-right-width: 0 !important;
+        }
+        table.body .btn table {
+          width: 100% !important;
+        }
+        table.body .btn a {
+          width: 100% !important;
+        }
+        table.body .img-responsive {
+          height: auto !important;
+          max-width: 100% !important;
+          width: auto !important;
+        }
+      }
+
+      .token_text {
+        font-weight: bold;
+      }
+
+      .token_text {
+        text-align: center;
+        color: #16395E;
+        font-size: 24px;
+      }
+
+    </style>
+  </head>
+  <body tal:define="portal view/get_portal; token view/get_token">
+    <table role="presentation" border="0" cellpadding="0" cellspacing="0" class="body">
+      <tr>
+        <td>&nbsp;</td>
+        <td class="container">
+          <div class="content">
+
+            <!-- START CENTERED WHITE CONTAINER -->
+            <table role="presentation" class="main">
+
+              <!-- START MAIN CONTENT AREA -->
+              <tr>
+                <td class="wrapper">
+                  <table role="presentation" border="0" cellpadding="0" cellspacing="0">
+                    <tr class="portal_title">
+                      <td>
+                        <h2>${portal/title}</h2>
+                      </td>
+                    </tr>
+                    <tr class="portal_description">
+                      <td>
+                        <h3 class="portal-description">${portal/description}</h3>
+                      </td>
+                    </tr>
+                    <tr class="token_header">
+                      <td>
+                        <div class="token_header">
+                          <h4>Il tuo codice per la conferma dell'indirizzo email:</h4>
```

Comment:
In english please, then add italian translation

Reply:
solved